supervisor/api/ingress.py

```diff
@@ -164,13 +164,15 @@ async def _handle_request(
         url = self._create_url(addon, path)
         source_header = _init_header(request, addon)
 
+        data = request.content if request.method == "POST" else await request.read()
```

Comment:
Add a comment above, why we do that

Reply:
```suggestion
		# Passing the raw stream breaks requests for some webservers 
		# since we just need it for POST requests really, for all other methods 
		# we read the bytes and pass that to the request to the add-on
        data = request.content if request.method == "POST" else await request.read()
```